Source/WebCore/animation/ViewTimeline.h

```diff
@@ -54,8 +54,8 @@ class ViewTimeline final : public ScrollTimeline {
     static Ref<ViewTimeline> createFromCSSValue(const Style::BuilderState&, const CSSViewValue&);
 
     Element* subject() const { return m_subject.get(); }
-    const CSSNumericValue& startOffset();
-    const CSSNumericValue& endOffset();
+    Ref<CSSNumericValue> startOffset();
+    Ref<CSSNumericValue> endOffset();
```

Comment:
I wonder if we should call these `computeStartOffset`/`computeEndOffset` since they're not trivial getters.

Reply:
These are API functions exposed via the IDL. Maybe we could do ImplementedAs to avoid confusion? Although we generally only do that to prefix functions with bindings.

Reply:
I'm fine either way, we could either use `[ImplementedAs]` or leave as is. I wonder if @cdumez or @graouts has opinions.

Reply:
Not a big deal to me, this looks OK to me as-is.